starknet_py/net/schemas/rpc.py

```diff
@@ -253,50 +256,85 @@ class TransactionV3Schema(TransactionSchema):
     )
 
 
-class DeprecatedInvokeTransactionSchema(DeprecatedTransactionSchema):
-    contract_address = Felt(data_key="contract_address", load_default=None)
-    sender_address = Felt(data_key="sender_address", load_default=None)
-    entry_point_selector = Felt(data_key="entry_point_selector", load_default=None)
+class InvokeTransactionV0Schema(DeprecatedTransactionSchema):
+    type = fields.Str(required=True, validate=validate.OneOf(["INVOKE"]))
```

Comment:
The same comment regarding `type` as [here](https://github.com/software-mansion/starknet.py/pull/1259/files#r1467612983)